src/sentry/static/sentry/app/components/avatar/avatarList.jsx

```diff
@@ -47,7 +47,7 @@ export default class AvatarList extends React.Component {
         {visibleUsers.map(user => {
           return (
             <StyledAvatar
-              key={user.id}
+              key={`${user.id}-${user.email}`}
```

Comment:
Like github commiter

Reply:
Can it ever be null? I guess what I'm getting at is, is `user.email` always unique for non-users? Since `user.id` is not, we need `user.email` to be globally unique in these cases.

Reply:
It definitely can, but I would rather fix this like this and see if there are cases where we're using an avatar list with users that have non unique ID or emails

Reply:
To be clear, this wasn't breaking anything since typically the avatar list wouldn't be re-rendering in these cases, but we were getting React warnings.

Reply:
Can we throw in ip address too (that's on non-users right?)

Reply:
Could the server responses include `slug` or `publicId` key that coalesces the various identifier sources we have? That would make this simpler and allow us to solve this problem in other places in the future.

Reply:
Yeah we probably could. I'd be happy to just merge this right now 🤷‍♂️